components/form/FormPartnersList.tsx

```diff
@@ -35,7 +35,7 @@ export const FormPartnersList: FC<FormPartnersListProps> = (props) => {
 
   options.push({
     value: 'none',
-    title: 'Outra - Entratemos em contato',
+    title: 'Outra - Entraremos em contato',
```

Comment:
@marceloglacial 
Shouldn't we have an English version of "Other" here?

Reply:
Good catch. You can use the dictionary pattern for it